src/util/boost-context/import.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
Can you add a comment here on when this file should be run? (I'm guessing when updating boost)

Reply:
Done